amazon-kinesis-client/src/test/java/software/amazon/kinesis/leases/dynamodb/DynamoDBLeaseRefresherTest.java

```diff
@@ -265,7 +266,40 @@ public void testLeaseTableExistsTimesOut() throws Exception {
     }
 
     @Test
-    public void testCreateLeaseTableTimesOut() throws Exception {
+    public void testCreateLeaseTableProvisionedBillingModeIfNotExists() throws Exception {
```

Comment:
Let's have more descriptive names for these tests - for example this test is testing behavior if the lease table already exists we should still return true when we catch the RNF exception.